doc_classes/OpenXRMetaColocationDiscovery.xml

```diff
@@ -0,0 +1,126 @@
+
+		func _on_discovery_result(advertisement_uuid: String, data: PackedByteArray):
+		    var session_info = JSON.parse_string(data.get_string_from_utf8())
+		    print("Found session: ", session_info)
+		[/codeblock]
+	</description>
+	<tutorials>
+		<link title="XR_META_colocation_discovery specification">https://registry.khronos.org/OpenXR/specs/1.1/html/xrspec.html#XR_META_colocation_discovery</link>
+	</tutorials>
+	<methods>
+		<method name="is_colocation_discovery_supported">
+			<return type="bool" />
+			<description>
+				Returns [code]true[/code] if the colocation discovery extension is supported on the current device, [code]false[/code] otherwise.
+				This should be checked before attempting to use advertisement or discovery features.
+			</description>
+		</method>
+		<method name="start_advertisement">
+			<return type="bool" />
+			<param index="0" name="buffer" type="PackedByteArray" />
+			<description>
+				Starts advertising the local session to nearby devices. The [param buffer] contains custom session data (max 1024 bytes) that will be transmitted to discovering devices.
+				Typically this data contains session information encoded as JSON, such as session ID, host name, and max players.
+				Returns [code]true[/code] if the advertisement request was successfully initiated, [code]false[/code] otherwise.
+				Listen to the [signal advertisement_started] signal to know when advertising actually begins, or [signal advertisement_failed] if it fails.
+				[b]Note:[/b] Only one advertisement can be active at a time. Call [method stop_advertisement] first if already advertising.
+			</description>
+		</method>
+		<method name="stop_advertisement">
+			<return type="bool" />
+			<description>
+				Stops the current advertisement. Returns [code]true[/code] if the stop request was successfully initiated, [code]false[/code] otherwise.
+				Listen to the [signal advertisement_stopped] signal to know when advertising actually stops.
+			</description>
+		</method>
+		<method name="start_discovery">
+			<return type="bool" />
+			<description>
+				Starts discovering nearby advertised sessions. Returns [code]true[/code] if the discovery request was successfully initiated, [code]false[/code] otherwise.
+				Listen to the [signal discovery_started] signal to know when discovery actually begins, and [signal discovery_result] for each discovered session.
+				[b]Note:[/b] Only one discovery operation can be active at a time. Call [method stop_discovery] first if already discovering.
+			</description>
+		</method>
+		<method name="stop_discovery">
+			<return type="bool" />
+			<description>
+				Stops the current discovery operation. Returns [code]true[/code] if the stop request was successfully initiated, [code]false[/code] otherwise.
+				Listen to the [signal discovery_stopped] signal to know when discovery actually stops.
+			</description>
+		</method>
+	</methods>
+	<signals>
+		<signal name="advertisement_started">
```

Comment:
For vendor extension signals the convention is to use the class name as prefix.
See https://godotvr.github.io/godot_openxr_vendors/api/class_openxrfbscenemanager.html#signals for an example.

Reply:
Thanks for the update!

Note that the prefix doesn't have to strictly duplicate the class name, just provide enough to distinguish so we can probably drop the `discovery` portion from the signals name.